### ISSUE TITLE
Fractions like 10/12 or 11/12 do not work

### ISSUE THREAD

**Issue by garytokyo** [colourgarden/avalanche#41]
I saw that this was previously a bug that was fixed however I can report that it doesn't work on my setup. I am using Laravel Mix to compile sass which is using `"node-sass": "^4.5.3"`, `"sass-loader": "^6.0.5"` and `"webpack": "^3.5.0"` (those would be the key packages that would effect this I would have thought).

I copied the `_avalanche.scss` directly into my code so I could debug it a bit - it seems that the class names are being generated correctly `DEBUG: \31\31\/12` (from line 138) but the `\` between the two 31s is missing when it is rendered as CSS so it becomes `.\3131\/12`. 

I am using MacOS Sierra and node v8.9.1

**Comment by garytokyo** [comment]
I think this could actually be down to css-loader or css-selector-tokenizer which the former uses.

**Comment by colourgarden** [comment]
Sounds like it could be something in your build step. Have you tried removing `sass-loader` and seeing if `node-sass` is spitting out the correctly escaped class names? To narrow down the problem.

You can also use the [`percentage` or `fragment` class name styles](https://github.com/colourgarden/avalanche/blob/master/_avalanche.scss#L15) to avoid the escaping issue.

**Comment by garytokyo** [comment]
Yeah I believe it is. I did a bit more research and it seems it is `css-selector-tokenizer` which is being used by `css-loader` I found an old closed ticket on the `css-loader` github talking about it stripping escape chars and someone said it was probably the tokenizer - which hasn't been updated in over a year so I guess it isn't going to be solved anytime soon!

I will close this then as nothing to do with Avalanche - unless you want to leave it in case other people have similar issue using it with webpack? Or maybe mention that in the Readme or something.
